File: CS/GridEntityFramework/App_Code/Model1.cs
using System;
using System.Collections.Generic;
using System.Data.Entity;
using System.Linq;
using System.Web;

namespace GridEntityFramework
{
    public partial class Model1 : DbContext
    {
        public Model1() : base("name=Model1")
        {
        }

        public virtual DbSet<Category> Categories { get; set; }
        public virtual DbSet<Product> Products { get; set; }

        protected override void OnModelCreating(DbModelBuilder modelBuilder)
        {
            modelBuilder.Entity<Product>().Property(e => e.UnitPrice).HasPrecision(19, 4);
        }
    }
}

File: VB/GridEntityFramework/App_Code/Model1.vb
Imports System.Data.Entity

Namespace GridEntityFramework

    Public Partial Class Model1
        Inherits DbContext

        Public Sub New()
            MyBase.New("name=Model1")
        End Sub

        Public Overridable Property Categories As DbSet(Of Category)

        Public Overridable Property Products As DbSet(Of Product)

        Protected Overrides Sub OnModelCreating(ByVal modelBuilder As DbModelBuilder)
            modelBuilder.Entity(Of Product)().Property(Function(e) e.UnitPrice).HasPrecision(19, 4)
        End Sub
    End Class
End Namespace
